.vale/styles/style_guide/Meilisearch.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Error: style_guide.Meilisearch.yml
+
+# Lists any possible typos for Meilisearch
```

Comment:
Would it be possible to add a scope for this rule? Just want to make sure we don't ever replace any `MeiliSearch`, `meiliSearch`, or potential future additions (e.g. `meilisearch`) that appear in code.

Reply:
Vale ignores anything in backticks and code blocks by default, so it won't be a problem